routes/index.go

```diff
@@ -145,17 +147,27 @@ func getFromAuth(path string) (*extractResponse, error) {
 // Middleware to handle InternalServerError
 func internalServerErrorHandler(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		fmt.Println("Inside Internal Server Middleware")
```

Comment:
can you remove this comment it would unnecessarily flood the logs for every request

Reply:
@kevkevinpal Sure

Reply:
@kevkevinpal Done.